### PR TITLE
issue: 847360 update github's README.md file

### PR DESCRIPTION
**Pull request by rosenbaumalex**
Add badges for:
1. Version
2. Coverity Scan

Signed-off-by: Alex Rosenbaum <Alexr@mellanox.com>

### CODE REVIEW

**Comment by rosenbaumalex** [comment]
You can see 'demo' of how this looks here: https://github.com/rosenbaumalex/libvma/tree/issue_847360_update_githubs_README_md_file#introduction

**Comment by rosenbaumalex** [comment]
this PR repalces https://github.com/Mellanox/libvma/pull/178

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/403/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/385/ for details (Mellanox internal link).

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[![GitHub version](https://badge.fury.io/gh/mellanox%2Flibvma.svg)](https://badge.fury.io/gh/mellanox%2Flibvma)
+[![Coverity Scan Build Status](https://scan.coverity.com/projects/8025/badge.svg)](https://scan.coverity.com/projects/libvma)
+
```
Comment (OphirMunk) [inline README.md]: @rosenbaumalex Are we guaranteed that forever we'll have converity pass? If we advance in version (and let's say coverity is compromised for some time) - what would happen?

Comment (rosenbaumalex) [inline README.md]: The badge is "ONLINE" similar to all other public activity in libmva

**Pull request merged**